consentconstants/tcf2/purposes.go

```diff
@@ -0,0 +1,91 @@
+package consentconstants
+
+import base "github.com/prebid/go-gdpr/consentconstants"
+
+// TCF 2.0 Purposes:
```

Comment:
All the "Regular" purposes found in https://iabeurope.eu/iab-europe-transparency-consent-framework-policies/ are listed. But I also noticed there's a part B "Special purposes" that lists two special purposes and I don't see them listed anywhere. Do we need them at all?

Reply:
We don't need them for this pass. We will need them in the future